test/resque_inline_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class ResqueInlineTest < Minitest::Test
+  def setup
+    Resque.inline = true
+    Resque.expects(:redis).never
+  end
+
+  def teardown
+    Resque.inline = false
+  end
+
+  def test_runs_inline
+    GoodJob.expects :perform
+    Resque.enqueue GoodJob
```

Comment:
Can you please add the brackets to the two `Resque.enqueue()` calls.

When calling a method I'd always opt for the bracketed version, unless its more of a DSL (e.g. like the testing framework).